pages/advanced/smart-account-bug-bounty.md

```diff
@@ -120,3 +120,9 @@ Public disclosure of the bug or the indication of an intention to exploit it on
 Any questions? Reach us via email ([bounty@safe.global](mailto:bounty@safe.global)) or [Discord](https://chat.safe.global). For more information on the Safe, check out our [website](https://safe.global) and our [GitHub](https://github.com/safe-global).
 
 _Happy hunting!_
+
+## Note on Safe{Wallet}
+
+Generally, bugs and issues regarding Safe{Wallet} frontend or backend are _out of scope_. This refers to the [web app](https://app.safe.global), mobile apps as well as backend services required. For any bug reports, please consider submitting an issue on the respective repository such as [safe-wallet-web](https://github.com/safe-global/safe-wallet-web/issues). 
+
+Please only use [bounty@safe.global](mailto:bounty@safe.global) for severe, security-related issues. We will carefully check all submissions however rewards remain voluntarily at our sole discretion.
```

Comment:
```suggestion
Please only use [bounty@safe.global](mailto:bounty@safe.global) for severe security-related issues. We will carefully check all submissions however rewards remain voluntarily at our sole discretion.
```

Can also be Either `for severe, security-related, issues` depending on your meaning.